packages/wasm/rollup.config.js

```diff
@@ -0,0 +1,16 @@
+const pkg = require('./package.json');
+const { builtinModules } = require('module');
+
+const dependencies = Object.keys(pkg.dependencies || {});
+
+export default [
+  {
+    input: 'src/index.js',
+    output: { exports: 'named', file: 'dist/index.js', format: 'cjs' },
+    external: [ ...builtinModules, ...dependencies ]
+  }
+  // {
```

Comment:
Are we intending to leave dead code here?

Reply:
no we can kill it